src/index.tsx

```diff
@@ -86,7 +86,7 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
 
     const fullHeight = isIos
       ? screenHeight
-      : screenHeight - (isIos ? StatusBar.currentHeight || 0 : 10);
+      : screenHeight - (StatusBar.currentHeight || 0);
```

Comment:
I didn't even notice I added isIos where it's already checked before.

Anyway, I did that by design because I find the Android margin too important. Could we expose that in a prop? I will be happy and not rechange again after merging your changes :P

Reply:
we can add a prop smth like `topOffset`? What do you think? By default it will be `ios: 0, adnroid: StatusBar.currentHeight || 0`

Reply:
I'm not sure about the name of the prop! `topMargin`???

Reply:
Yeah, I'm fine with these default value. I would say `modalTopOffset`

Reply:
OK, will do